src/comps/Deposit.tsx

```diff
@@ -306,9 +307,10 @@ const DepositFlowConfirm = ({
         bitcoinTxOutputIndex: 0,
         reclaimScript: reclaimScriptHex,
         depositScript: depositScriptHexPreHash,
-        url: emilyUrl,
       };
 
+      console.log({ emilyReqPayloadClient: emilyReqPayload });
+
```

Comment:
nit: can we also log this (with empty `bitcoinTxid`) before calling leather/xverse? For the current issue we are having we would not get any log

Reply:
done